packages/AlgernonTests.package/ALGTestUI.class/instance/testAlgernonPositioningWindowIsSameSizeAfterReopening.st

```diff
@@ -0,0 +1,6 @@
+testing
+testAlgernonPositioningWindowIsSameSizeAfterReopening
```

Comment:
Maybe we should move Algernon beforehand ?

Reply:
Did that.